src/test/java/integrationtestclasses/MeetingControllerIT.java

```diff
@@ -0,0 +1,120 @@
+
+        HttpEntity<String> httpEntity =
+                new HttpEntity<String>(scheduleMeetingRequestString, headers);
+        ResponseEntity<Response> responseEntity = restTemplate.exchange(createURLWithPort("/meeting"), HttpMethod.POST, httpEntity,
+                Response.class);
+        assertEquals(201, responseEntity.getStatusCodeValue());
+        assertNotNull(responseEntity.getBody());
+        assertNotNull(responseEntity.getBody().getData());
+        System.out.println(responseEntity.getBody().getData());
+    }
+
+    @Test
+    public void scheduleMeetingFailTestRoomIsGiven() throws JsonProcessingException {
+        HttpHeaders headers = new HttpHeaders();
+        headers.setContentType(MediaType.APPLICATION_JSON);
+
+        LocalDate dateOfMeeting = LocalDate.of(2022,8,26);
+        LocalTime startTime = LocalTime.of(16,00);
+        LocalTime endTime = LocalTime.of(16,50);
+        List<String> employeeList = Arrays.asList("abc-11", "abc-12", "abc-13", "abc-14", "abc-15", "abc-16");
+        AddMeetingDataRequest request = new AddMeetingDataRequest
+                ("abc-11" , "sync-up","details",employeeList, dateOfMeeting,startTime, endTime, "Room5");
+
+        String scheduleMeetingRequestString = objectMapper.writeValueAsString(request);
+
+        HttpEntity<String> httpEntity =
+                new HttpEntity<String>(scheduleMeetingRequestString, headers);
+
+        ResponseEntity<Response> responseEntity = restTemplate.exchange(createURLWithPort("/meeting"), HttpMethod.POST, httpEntity,
+                Response.class);
+
+        assertEquals(400, responseEntity.getStatusCodeValue());
+        assertNotNull(responseEntity.getBody());
+        assertEquals("meeting room does not exist or is closed" , responseEntity.getBody().getError());
+    }
+
+    @Test
+    public void scheduleMeetingFailTestRoomIsNotGiven() throws JsonProcessingException {
+        HttpHeaders headers = new HttpHeaders();
+        headers.setContentType(MediaType.APPLICATION_JSON);
+        LocalDate dateOfMeeting = LocalDate.of(2022,8,26);
+        LocalTime startTime = LocalTime.of(16,00);
+        LocalTime endTime = LocalTime.of(16,50);
+        List<String> employeeList = Arrays.asList("abc-11", "abc-12", "abc-13", "abc-14", "abc-15", "abc-16" ,"abc-17");
+        AddMeetingDataRequest request = new AddMeetingDataRequest
+                ("abc-11" , "sync-up","details",employeeList, dateOfMeeting,startTime, endTime, "");
+
+        String scheduleMeetingRequestString = objectMapper.writeValueAsString(request);
+
+        HttpEntity<String> httpEntity =
+                new HttpEntity<String>(scheduleMeetingRequestString, headers);
+
+        ResponseEntity<Response> responseEntity = restTemplate.exchange(createURLWithPort("/meeting"), HttpMethod.POST, httpEntity,
+                Response.class);
+
+        assertEquals(400, responseEntity.getStatusCodeValue());
```

Comment:
we should write the test description on why we are excepting 400 here, there is an annotation called @DisplayName